apps/client/src/features/viewers/minimal-timer/MinimalTimer.tsx

```diff
@@ -153,27 +158,27 @@ export default function MinimalTimer(props: MinimalTimerProps) {
     >
       <NavigationMenu />
       {!hideMessagesOverlay && (
+        <div className={showOverlay ? 'message-overlay message-overlay--active' : 'message-overlay'}>
+          <div className='message'>{pres.text}</div>
+        </div>
+      )}
+      {time.finished && general.endMessage && !hideEndMessage ? (
```

Comment:
super nitpick:
I would prefer having "complex" logic away from the presentation.
This would mean extracting this to a variable before the render method, next to the other stuff
like 
`const showEndMessage = time.current < 0 && general.endMessage && !hideEndMessage ;`

Also, note I am using `time.current < 0 `instead of `time.finished`.
`time.finished `was a lazy shortcut from the early days that I came to regret and would like to get rid of it :D